examples/fishjam-chat/package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "prebuild": "expo prebuild",
     "start": "expo start",
-    "android": "expo run:android",
-    "ios": "expo run:ios",
+    "android": "expo run:android -d",
+    "ios": "expo run:ios -d",
```

Comment:
**question:** What does `-d` do? And why do we need it?

Reply:
it allows to select device to run app, including real devices.

Reply:
:50_50:
Can we leave it to developer to add this param? I usually prefer to have run app on default device, and use `yarn ios -d` if I want to change it.

Reply:
:50_50:
Can we leave it to developer to add this param? I usually prefer to have run app on default device, and use yarn ios -d if I want to change it.

Reply:
i will change it